xml/admin_ceph_datamgm.xml

```diff
@@ -1149,9 +1149,9 @@ rule example2 {
     based on the number of PGs in the defined pool and the ID of the pool.
    </para>
    <para>
-    The object’s contents within a placement group are stored in a set of
-    OSDs. For example, in a replicated pool of size two, each placement group
-    will store objects on two OSDs:
+    The object’s contents within a placement group are stored in a set of OSDs.
```

Comment:
The curly quote here _is_ **technically** correct but I think we should leave smart quotation marks up to DAPS. I suggest a global search-and-replace with simple straight quotation marks or apostrophes – that is, ['] and not the back-tick [`].